lib/extract_field/credentials_intelligence/V2CredentialsIntelligenceProtocol.js

```diff
@@ -0,0 +1,50 @@
+const { ICredentialsIntelligenceProtocol } = require('./ICredentialsIntelligenceProtocol');
+const { LoginCredentialsFields } = require('../../models/LoginCredentialsFields');
+const pxUtil = require('../../pxutil');
+const pxConstants = require('../../utils/constants')
```

Comment:
When possible I prefer to import only the specific functions / objects I need rather than the entire module.